libtsuba/src/FileView.cpp

```diff
@@ -40,21 +40,20 @@ FileView::~FileView() {
 
 katana::Result<void>
 FileView::Unbind() {
-  katana::Result<void> res = katana::ResultSuccess();
   if (valid_) {
     // Resolve all outstanding reads so they don't write to the memory we are
     // about to unmap
     if (auto res = Resolve(0, file_size_); !res) {
-      return res.error();
+      return res.error().WithContext("resolving for unmap");
     }
```

Comment:
I'm just learning by example rather than reading the docs, so feel free to ignore.  When you have an error code and message, you use `KATANA_ERROR`, but when you don't have an error code, you append a `WithContext `message.